CHANGELOG.rst

```diff
@@ -8,6 +8,19 @@ Version History
 
 - `BlockingConnection.consume` now returns `(None, None, None)` when inactivity timeout is reached (`PR <https://github.com/pika/pika/pull/899>`_)
 
+API changes from previous versions:
+
+pika/channel.py
+
+def basic_consume(self,
+                  queue,
+                  on_message_callback,
+                  auto_ack=False,
+                  exclusive=False,
+                  consumer_tag=None,
+                  arguments=None,
+                  callback=None)
```

Comment:
@vitaly-krugl how would you like API changes documented? I can't really think of a good way, except maybe having old version next to the new one in a table, perhaps.

Reply:
> except maybe having old version next to the new one in a table, perhaps

@lukebakken - this sounds reasonable